package.json

```diff
@@ -3,28 +3,29 @@
   "private": true,
   "workspaces": [
     "packages/@tinacms/*",
+    "deprecated-packages/*",
```

Comment:
I wonder if we're better off just leaving this out. By including it we sort of have to opt-out of everything that might run those packages (as you've done for some of the scripts below). Seems like it will work but at the same time I'm not sure what we gain by including it.

Reply:
👍  Yeah this is a much better way of doing it